spec.bs

```diff
@@ -803,23 +804,24 @@ An interest group is a [=struct=] with the following items:
 :: Null or an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] are
   {{double}}. Overrides the {{AuctionAdConfig}}'s corresponding priority signals.
 : <dfn>execution mode</dfn>
-:: A [=string=], defaulting to "compatibility".
+:: A [=string=], defaulting to "compatibility". Acceptable values are "compatibility" and "group-by-origin".
 : <dfn>bidding url</dfn>
-:: Null or a [=URL=]. The URL to fetch the buyer's JavaScript from.
+:: Null or a [=URL=]. The URL to fetch the buyer's JavaScript from. Must be of origin `owner`.

```

Comment:
What does the last sentence here mean? Can we say `It's [=url/origin=] will always be [=same origin=] with [=owner=] (or however we'd link to owner here)`

Reply:
Do we normally list the restrictions of each field in the structure definition, or check the restrictions in the method steps that set the fields? Wondering if there's a best practice thing.
For example, we check if field originA is same origin with originB when setting originA in the method, then do we also say "originA must be same origin with originB" in field originA's definition here?

Reply:
Yeah good point. We should probably make this a note (with the text I describe above) directly below the member, in addition to the actual checks that we perform in real algorithms.

Reply:
Done

Reply:
So, I'm getting markup validation errors because it doesn't like having a Note: inside a list, I think? How do I fix this?

Reply:
Fixed now, thanks for the help :)